tests/test_cliPluginLoader.py

```diff
@@ -74,12 +75,39 @@ def duplicate_command_test_env(runner):
             yield
 
 
+class FailedLoadTest(unittest.TestCase):
+
+    def setUp(self):
+        # The caplog fixture is a pytest feature to manipulate and read log
+        # output from the test.
```

Comment:
Does this mean log output from the logging package? We don't use pytest fixtures for that, we use unittest:
```python
with self.assertLogs(level="INFO") as cm:
   ...
```

Reply:
I was having trouble capturing logging for testing when testing with pytest, because it captures logs which is why I was using the pytest fixture. But this solution works better, does not require special casing.